Userland/Libraries/LibGL/Shaders/Shader.h

```diff
@@ -26,6 +28,7 @@ class Shader final : public RefCounted<Shader> {
     ErrorOr<void> compile();
     GLenum type() const { return m_type; }
     bool compile_status() const { return m_compile_status; }
+    GLSL::ObjectFile const* object_file() const { return m_object_file.ptr(); }
```

Comment:
Was this added in my original PR? Looks weird, maybe returning a const reference would be nicer.

Reply:
Not sure, I will fix this otherwise.

Reply:
On second thought, I don't think we need to change this. Everything else that uses this function assumes it will get a pointer, and it's not guaranteed that you will have a valid reference to the `ObjectFile`.

Reply:
Right, I think I did it like that because of the `ObjectFile` vector in `GL::Program::link()`. I remember this was done on purpose.